_monitoring-your-cluster/pa/rca/shard-hotspot.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: default
+title: Hot shard identification
+parent: Root Cause Analysis
+grand_parent: Performance Analyzer
+nav_order: 30
+---
+
+## Hot shard identification
```

Comment:
Do you want to begin this topic with a heading level 1? Consider that "Response fields" further down is the same heading level.

Reply:
Thank you! Changed.